jetty-rewrite/src/main/java/org/eclipse/jetty/rewrite/handler/RuleContainer.java

```diff
@@ -26,14 +26,15 @@
 import org.eclipse.jetty.server.Request;
 import org.eclipse.jetty.util.ArrayUtil;
 import org.eclipse.jetty.util.URIUtil;
+import org.eclipse.jetty.util.component.Dumpable;
 import org.eclipse.jetty.util.log.Log;
 import org.eclipse.jetty.util.log.Logger;
 
 /**
  * Base container to group rules. Can be extended so that the contained rules
  * will only be applied under certain conditions
  */
-public class RuleContainer extends Rule
+public class RuleContainer extends Rule implements Dumpable
```

Comment:
Seems unrelated

Reply:
yep - was needed to help debug these changes, but is ultimately unrelated.   I will move it to a PR against 9.4 as it is useful there as well.